### ISSUE TITLE
DevTools window makes AS freeze completely

### ISSUE THREAD

**Issue by deakjahn** [flutter/flutter-intellij#7597]
## Steps to Reproduce

1. Open Android Studio
2. Locate the new DevTools window (or add the icon from the three-point menu)
3. Try to start DevTools
4. The window appears with "Nothing to show" and Android Studio freezes completely. Completely unresponsive, the only way out is to kill if from Windows Task Manager.

## Version info

Please paste the output of running `flutter doctor -v` here (available from the command
line or from `Tools > Flutter > Flutter Doctor`). It will provide the version of the
Flutter framework as well as of the IntelliJ plugin.

P:\Flutter\bin\flutter.bat doctor --verbose
[√] Flutter (Channel stable, 3.24.0, on Microsoft Windows [Version 10.0.22631.3880], locale en-US)
    • Flutter version 3.24.0 on channel stable at P:\Flutter
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision 80c2e84975 (8 days ago), 2024-07-30 23:06:49 +0700
    • Engine revision b8800d88be
    • Dart version 3.5.0
    • DevTools version 2.37.2

[√] Windows Version (Installed version of Windows is version 10 or higher)

[!] Android toolchain - develop for Android devices (Android SDK version 34.0.0)
    • Android SDK at P:/Android_SDK
    X cmdline-tools component is missing
      Run `path/to/sdkmanager --install "cmdline-tools;latest"`
      See https://developer.android.com/studio/command-line for more details.
    X Android license status unknown.
      Run `flutter doctor --android-licenses` to accept the SDK licenses.
      See https://flutter.dev/to/windows-android-setup for more details.

[X] Chrome - develop for the web (Cannot find Chrome executable at .\Google\Chrome\Application\chrome.exe)
    ! Cannot find Chrome. Try setting CHROME_EXECUTABLE to a Chrome executable.

[!] Visual Studio - develop Windows apps (Visual Studio Community 2022 17.10.5)
    • Visual Studio at C:\Program Files\Microsoft Visual Studio\2022\Community
    • Visual Studio Community 2022 version 17.10.35122.118
    • Windows 10 SDK version 10.0.22621.0
    X Visual Studio is missing necessary components. Please re-run the Visual Studio installer for the "Desktop development with C++" workload, and include these components:
        MSVC v142 - VS 2019 C++ x64/x86 build tools
         - If there are multiple build tool versions available, install the latest
        C++ CMake tools for Windows
        Windows 10 SDK

[√] Android Studio (version 2024.1)
    • Android Studio at C:\Program Files\Android\Android Studio
    • Flutter plugin can be installed from:
       https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
       https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 17.0.11+0--11852314)

[√] Connected device (2 available)
    • Windows (desktop) • windows • windows-x64    • Microsoft Windows [Version 10.0.22631.3880]
    • Edge (web)        • edge    • web-javascript • Microsoft Edge 127.0.2651.86

[√] Network resources
    • All expected network resources are available.

! Doctor found issues in 3 categories.
Process finished with exit code 0

**Comment by helin24** [comment]
@deakjahn can you send your logs to my email? The logs are located at Help > Show log in finder > File called `idea.log`. My email is helinx@google.com.

**Comment by deakjahn** [comment]
@helin24 As I said above, there's no log. I mean, there is one if I normally use the program but if I delete the log, go fresh into AS, cause it to freeze immediately and kill it from Windows, there will be no `idea.log` written, the program probably doesn't get any chance to flush it. The only thing that appears is the folder above.

**Comment by helin24** [comment]
Sorry I missed that! I'm looking at the thread dumps now. 

Are you able to remove all of the Flutter windows from your sidebar, and then only open the Flutter inspector?
<img width="243" alt="Screenshot 2024-08-08 at 10 54 55 AM" src="https://github.com/user-attachments/assets/e68794c6-a361-4a6f-90be-13d581a636bd">

The process for opening the inspector differs a bit from the other tool windows, so I'm wondering if this one also crashes.

**Comment by deakjahn** [comment]
Yes, that works. With no running application it does nothing but it doesn't crash, either.

**Comment by TechnicJelle** [comment]
I'm having the same issue. My current workaround is to do `flutter run` in the terminal, and just use the inspector from Firefox, instead of integrated in Android Studio...
Should I send a log file, too?
The ones I can find here are rather large...

**Comment by pq** [comment]
This one is fixed right @helin24?

**Comment by helin24** [comment]
I'm not sure if this was fixed, but it's most likely obsolete now since we've changed how DevTools starts. I'll close this for now; if it's still a problem, we can open a new issue.

**Comment by deakjahn** [comment]
@helin24 It froze just as well with Narwhal but Patch 1 seems to have helped.
